lib/run-test-with-retry.js

```diff
@@ -49,11 +51,15 @@ module.exports = class RunTestWithRetry {
       results.push(result)
     }
 
+    const success = results.some(x => x.success)
+
+    success && this.eventsEmitter.emit('run-step.success', { testId, stepName })
```

Comment:
self minification is the best minification

Reply:
Hi Arjan :)